src/SingleDatePicker.elm

```diff
@@ -87,24 +99,26 @@ type Status
 {-| A record of default settings for the date picker. Extend this if
 you want to further customize the date picker.
 
-Requires a `msg` that expects a tuple containing a datepicker instance
-and a `Maybe Posix` representing a selected datetime.
+Requires a `Zone` to inform the picker in which time zone it should
+display the picked time as well as a `msg` that expects a tuple containing a
+datepicker instance and a `Maybe Posix` representing a selected datetime.
 
     ( DatePicker, Maybe ( Posix, Posix ) ) -> msg
 
 -}
-defaultSettings : (( DatePicker, Maybe Posix ) -> msg) -> Settings msg
-defaultSettings internalMsg =
+defaultSettings : Zone -> (( DatePicker, Maybe Posix ) -> msg) -> Settings msg
+defaultSettings zone internalMsg =
     { formattedDay = Utilities.dayToNameString
     , formattedMonth = Utilities.monthToNameString
     , focusedDate = Nothing
     , dateTimeProcessor =
-        { isDayDisabled = always False
-        , allowedTimesOfDay = \_ -> { startHour = 0, startMinute = 0, endHour = 23, endMinute = 59 }
+        { isDayDisabled = \_ _ -> False
+        , allowedTimesOfDay = \_ _ -> { startHour = 0, startMinute = 0, endHour = 23, endMinute = 59 }
         }
     , internalMsg = internalMsg
-    , dateStringFn = \_ -> ""
-    , timeStringFn = \_ -> ""
+    , dateStringFn = \_ _ -> ""
+    , timeStringFn = \_ _ -> ""
+    , zone = zone
```

Comment:
Why does the user has to provide a zone in order to use the default-settings? Why not just default to UTC?

**Or even better..** let the user define the timezone of the datetime-picker in the `init`, since the timezone seems to be relevant for the `update` and `view` (` SingleSelect.init Time.utc`)